<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmConfig</class>
 <widget class="QDialog" name="frmConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>492</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_title" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="lab_Ico">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lab_Title">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>系统设置</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_menu" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="btnMenu_Close">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>ArrowCursor</cursorShape>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_main" native="true">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="gboxLeft">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>0</height>
         </size>
        </property>
        <property name="title">
         <string/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="btnBase">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>基本设置</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnUser">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>操作员管理</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnTime">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>时间设置</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnNet">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>网络设置</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnOther">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>其他设置</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSave">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>保存配置</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="gboxBase">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="6" column="0">
          <widget class="QLabel" name="labBaudRate">
           <property name="text">
            <string>波特率:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QComboBox" name="cboxTimeout"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="labHeartInterval">
           <property name="text">
            <string>心跳间隔:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="labConnectMode">
           <property name="text">
            <string>通信模式:</string>
           </property>
          </widget>
         </item>
         <item row="10" column="0">
          <widget class="QLabel" name="labServerIP">
           <property name="text">
            <string>远程IP:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="12" column="2">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="labPortName">
           <property name="text">
            <string>串口号:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="labListenPort">
           <property name="text">
            <string>本地端口:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="labTimeout">
           <property name="text">
            <string>超时时长:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QComboBox" name="cboxHeartInterval"/>
         </item>
         <item row="2" column="3">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>秒</string>
           </property>
          </widget>
         </item>
         <item row="11" column="0">
          <widget class="QLabel" name="labServerPort">
           <property name="text">
            <string>远程端口:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="labAppStyle">
           <property name="text">
            <string>界面样式:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>秒</string>
           </property>
          </widget>
         </item>
         <item row="11" column="2">
          <widget class="QLineEdit" name="txtServerPort"/>
         </item>
         <item row="10" column="2">
          <widget class="QLineEdit" name="txtServerIP"/>
         </item>
         <item row="9" column="2">
          <widget class="QLineEdit" name="txtListenPort"/>
         </item>
         <item row="6" column="2">
          <widget class="QComboBox" name="cboxBaudRate">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QComboBox" name="cboxPortName">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QComboBox" name="cboxConnectMode">
           <item>
            <property name="text">
             <string>Com_RS232</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Tcp_Client</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Tcp_Server</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QComboBox" name="cboxAppStyle"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labAppInput">
           <property name="text">
            <string>输入法:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QComboBox" name="cboxAppInput">
           <item>
            <property name="text">
             <string>开启</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>关闭</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="gboxUser">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QListWidget" name="listUser">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="gboxUserInfo">
           <property name="title">
            <string/>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="1" column="0">
             <widget class="QLabel" name="labUserName">
              <property name="text">
               <string>操作员名称:</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="labUserRole">
              <property name="text">
               <string>操作员权限:</string>
              </property>
             </widget>
            </item>
            <item row="8" column="0" colspan="2">
             <widget class="QPushButton" name="btnUpdate">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="text">
               <string>修改</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="labLineName">
              <property name="text">
               <string>生产线名称:</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLineEdit" name="txtLineName">
              <property name="text">
               <string>321</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="txtUserName">
              <property name="text">
               <string>123</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="labUserID">
              <property name="text">
               <string>操作员工号:</string>
              </property>
             </widget>
            </item>
            <item row="7" column="0" colspan="2">
             <widget class="QPushButton" name="btnDelete">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="text">
               <string>删除</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QComboBox" name="cboxUserRole"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="labUserPwd">
              <property name="text">
               <string>操作员密码:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="labLineID">
              <property name="text">
               <string>生产线编号:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QComboBox" name="cboxLineID"/>
            </item>
            <item row="6" column="0" colspan="2">
             <widget class="QPushButton" name="btnAdd">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="text">
               <string>添加</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="txtUserPwd">
              <property name="text">
               <string>123</string>
              </property>
              <property name="echoMode">
               <enum>QLineEdit::Password</enum>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="cboxUserID"/>
            </item>
            <item row="9" column="0" colspan="2">
             <widget class="QPushButton" name="btnUpdatePwd">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="text">
               <string>更改密码</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="gboxOther">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cboxAppStyle</tabstop>
  <tabstop>cboxHeartInterval</tabstop>
  <tabstop>cboxTimeout</tabstop>
  <tabstop>cboxConnectMode</tabstop>
  <tabstop>cboxPortName</tabstop>
  <tabstop>cboxBaudRate</tabstop>
  <tabstop>txtListenPort</tabstop>
  <tabstop>txtServerIP</tabstop>
  <tabstop>txtServerPort</tabstop>
  <tabstop>listUser</tabstop>
  <tabstop>cboxUserID</tabstop>
  <tabstop>txtUserName</tabstop>
  <tabstop>txtUserPwd</tabstop>
  <tabstop>cboxLineID</tabstop>
  <tabstop>txtLineName</tabstop>
  <tabstop>cboxUserRole</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
